data/processed/gtr/total_gtr_projects_stem.yaml

```diff
@@ -0,0 +1,24 @@
+api: the type of API to be used to fetch data: gtr.ukri (but processed by Nesta and available from DAPS)
+api_documentation: https://github.com/nestauk/gtr_data_processing
+description: This dataset contains information about research projects funded by UK Research Councils and Innovate UK since ~ 2006. We have combined project and organisation data to analyse geographical activity and labelled projects with their probably discipline based on a semantic description of their abstract. 
+endpoint_source: Nesta DAPS
+enpoint_static: Nesta data_getters lab
+framework_group: 1
+is_experimental: True
+nuts_level: 2
+source: the data provider: UKRI
+data_date: 20200204
+year_range: [2006, 2018]
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: Total number of projects led by organisations in the NUTS area in STEM subjects (see aux/gtr_stem_disciplines for the stem disciplines we are considering)
+    id: total_gtr_projects_stem
+    label: Number of projects in STEM disciplines led by organisations in the area
+    type: total_projects
```

Comment:
What `type` is this? Where is this defined?

If it is just a count of items, instead of `type: total_projects` please just use `data_type: int`.